sentences/en/light_HassTurnOff.yaml

```diff
@@ -18,18 +18,12 @@ intents:
           - "<area> <light> (off|out)"
           - "<light> (off|out) [in] <area>"
           - "<turn> (off|out) [<all>] <area> <light>"
-          - "<turn> (off|out) [<all>] <light> <in> <area>"
+          - "<turn> (off|out) [<all>] [<the>] <light> in <area>"
```

Comment:
<all> is already (all[[ of] <the>]|every[ single]|each[ and every]). Why would you add [<the>] as an extra? You could also get a match for every the light, which you don't want